# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1241467106 14400
#      Mon May 04 15:58:26 2009 -0400
# Node ID 07966ebb1055060d0124330e77d54f0788ab3fb2
# Parent  224634b2e6e3a7b81035cd557be062564966f9db
contributors.in: add Peter O'Gorman to the list

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -141,16 +141,17 @@ Victor Munoz
 Carmen Navarrete
 Todd Neal
 Al Niessner
 Rick Niles
 Takuji Nishimura
 Eric Norum
 Krzesimir Nowak
 Michael O'Brien
+Peter O'Gorman
 Thorsten Ohl
 Arno Onken
 Luis F. Ortiz
 Scott Pakin
 Gabriele Pannocchia
 Sylvain Pelissier
 Per Persson
 Primozz Peterlin
